WordPress/src/main/java/org/wordpress/android/ui/reader/views/compose/filter/ReaderFilterChipGroup.kt

```diff
@@ -113,20 +127,6 @@ fun ReaderFilterChipGroup(
         AnimatedVisibility(visible = isBlogChipVisible && isTagChipVisible) {
             Spacer(Modifier.width(Margin.Medium.value))
         }
```

Comment:
🚫 This spacer should be between the 2 chips, just move these lines after line 111 and it should be fine!

Reply:
Thanks for catching that! I've pushed this fix.